src/osx/cocoa/combobox.mm

```diff
@@ -160,6 +160,11 @@ - (void)comboBoxSelectionDidChange:(NSNotification *)notification
 }
 @end
 
+NSComparator comp = ^(id obj1, id obj2)
```

Comment:
Can this be defined locally, i.e. inside the (only) function which uses it? If not, it should be given some more unique name, having `comp` as a global name is a really bad idea.